src/main/resources/assets/tconstruct/book/materials_and_you/en_us/intro/tinkerstation.json

```diff
@@ -0,0 +1,16 @@
+{
+  "title":"Tinker Station",
+  "text": [
+    {
+      "text": "The first step in your grand tinkering adventure is crafting your first tool. It may not start off the best, but it is your work, and you should be proud of it."
+    },
+    {
+      "text": "Once you have acquired all the parts you need to make a tool, construction can begin! Make sure you have a Tinker Station made in order to make them, select which tool you want to make and tinker away! ",
+      "paragraph": true
+    },
+	{
+      "text": "You can also use the Tinker Station to add modifiers to your tools. The basic table only has enough room to combine three parts maximum. Other Master Tinkers can tell you more.",
```

Comment:
I think I would expand to "only has enough room to combine three parts maximum, meaning no broad tools can be crafted". The last bit is also a little awkward, perhaps something like "Other master tinkers know the secret to making tools out of four or five parts"